bin/build.sh

```diff
@@ -1,57 +1,71 @@
 #!/bin/bash
 
+# Makes it so that all commands are displayed as they get executed
 set -x
 
 wca_url="https://worldcubeassociation.org/"
+stylesheet=$(realpath "./assets/style.css")
+
+# Remove potentially cached PDFs from the last build run and make an empty build folder
+rm -rf build
+mkdir build
+# Copy source folders into build
+cp -r documents/ build/
+cp -r edudoc/ build/
+# Copy WCA logo into build/assets temporarily
+mkdir build/assets
+cp assets/WCAlogo_notext.svg build/assets/
```

Comment:
What about jsut copying the entire `assets` folder into `build` and not only the logo?
That way you could also reference the CSS stylesheets from within the build folder rather than copying them around later manually.

Reply:
@gregorbg this would not work, as every single HTML file references the stylesheet and because HTML doesn't understand absolute paths, it's much better to just copy the stylesheet into each folder temporarily and have the HTML reference it directly. In fact, I haven't found any way around this. Keep in mind that this is exactly what we did with the edudoc header to begin with, and all of these files get deleted in the end anyways, so I think it should be okay.

Reply:
I just did a real basic test that reveals HTML _can_ handle absolute paths.